src/current/cockroachcloud/upgrade-policy.md

```diff
@@ -9,34 +9,43 @@ This page describes the support and upgrade policy for clusters deployed in Cock
+{{site.data.alerts.end}}
+
+### Version and upgrade support by cloud plan
+
+[Major versions]({% link releases/index.md %}) of CockroachDB are labeled either as [Regular releases]({% link releases/index.md %}#major-releases) or [Innovation releases]({% link releases/index.md %}).
 
 - **Regular releases** are supported for 12 months from their initial production release date.
 - **Innovation releases** are supported for 6 months from their initial production release date.
 
-For each release type, the end date of this period is called End of Support (EOS).
+CockroachDB {{ site.data.products.basic }} and {{ site.data.products.standard }} clusters only support **Regular releases**. This ensures that these clusters are using versions of CockroachDB that receive a full year of support with patches, offering long-term stability. All CockroachDB {{ site.data.products.basic }} and {{ site.data.products.standard }} clusters are automatically upgraded to the next major regular release upon its GA release.
 
-A cluster running an unsupported CockroachDB version is not eligible for Cockroach Labs’ [availability SLA](https://www.cockroachlabs.com/cloud-terms-and-conditions/cockroachcloud-technical-service-level-agreement/).
-
-All CockroachDB {{ site.data.products.basic }} clusters, and CockroachDB {{ site.data.products.standard }} clusters by default, will automatically be upgraded to the next major version while the current one is still supported, to prevent the cluster from reaching EOS.
+{{site.data.alerts.callout_info}}
+CockroachDB {{ site.data.products.standard }} clusters have the option to [disable automatic major-version upgrades]({% link cockroachcloud/cluster-management.md %}#manage-cluster-upgrades). In this case, you are responsible for upgrading the cluster before the current version hits the end-of-support (EOS) date in order to maintain uninterrupted support and SLA guarantees. Clusters that have not upgraded for one year may be upgraded automatically.
+{{site.data.alerts.end}}
 
-- If you [disable automatic major-version upgrades for CockroachDB {{ site.data.products.standard }}]({% link cockroachcloud/cluster-management.md %}#manage-cluster-upgrades), you are responsible for upgrading the cluster before its version reaches its EOS date to maintain uninterrupted support and SLA guarantees. Clusters that have not upgraded for one year may be upgraded automatically.
-- You are responsible for upgrading a CockroachDB {{ site.data.products.advanced }} cluster before its current version reaches its EOS date to maintain uninterrupted support and SLA guarantees.
+CockroachDB {{ site.data.products.advanced }} clusters optionally support **Innovation releases** as well as regular releases, allowing users to access the latest CockroachDB features on an accelerated timeline, while still receiving 6 months of patch support before a recommended upgrade to another major version.
```

Comment:
Please refer to the above comment on aligning maintenance support on SH. The way its currently worded is not clear